app/controllers/api/v1/services_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::ServicesController < ApplicationController
+  def index
+    services = Service.all.order('created_at DESC')
+    render json: { services: services }.to_json
+  end
+
+  def create
+    @service = Service.new(service_params)
+
+    if @service.save
+      render json: { message: 'Service has been created' }.to_json
+    else
+      render json: { message: 'Service was not created' }.to_json
+    end
+  end
+
+  def delete
+    @service = Service.find(params[:id])
+    if @service.destroy
+      render json: { message: 'Service has been deleted' }.to_json
+    else
+      render json: { message: 'No action was taken' }.to_json
+    end
+  end
```

Comment:
`DELETE api/v1/services/:id` is being routed to `ServicesController#destroy`, not to `ServicesController#delete`. Please, rename this function.